src/components/wrapper/wrapper.js

```diff
@@ -1,43 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { useApolloQuery } from '../../apollo/utils';
+import { client } from '../../apollo/config';
+import { GraphQLProvider } from '../provider/provider';
+import { GET_VERSIONS } from '../../apollo/queries';
 import { isLoading } from '../../selectors/loading';
 import classnames from 'classnames';
 import GlobalToolbar from '../global-toolbar';
 import FlowChartWrapper from '../flowchart-wrapper';
 import ExperimentWrapper from '../experiment-wrapper';
 import SettingsModal from '../settings-modal';
-import { Provider } from '../provider/provider';
+import UpdateReminder from '../update-reminder';
 
 import './wrapper.css';
 
 /**
  * Main app container. Handles showing/hiding the sidebar nav, and theme classes.
  */
-export const Wrapper = ({ theme }) => (
-  <Provider useMocks={false}>
-    <div
-      className={classnames('kedro-pipeline kedro', {
-        'kui-theme--dark': theme === 'dark',
-        'kui-theme--light': theme === 'light',
-      })}
-    >
-      <h1 className="pipeline-title">Kedro-Viz</h1>
-      <Router>
-        <GlobalToolbar />
-        <SettingsModal />
-        <Switch>
-          <Route exact path={['/', '/flowchart']}>
-            <FlowChartWrapper />
-          </Route>
-          <Route path={['/runsList', '/runsList/:id']}>
-            <ExperimentWrapper />
-          </Route>
-        </Switch>
-      </Router>
-    </div>
-  </Provider>
-);
+export const Wrapper = ({ theme }) => {
+  const { data } = useApolloQuery(GET_VERSIONS, { client });
+  const [dismissed, setDismissed] = useState(false);
+  const [isOutdated, setIsOutdated] = useState(false);
+  const [latest, setLatest] = useState(null);
```

Comment:
I said this is another comment: it could be nice to make `latest` more specific.